src/transformers/tokenization_bert.py

```diff
@@ -173,6 +176,7 @@ def __init__(
         cls_token="[CLS]",
         mask_token="[MASK]",
         tokenize_chinese_chars=True,
+        strip_accents=None,
```

Comment:
I don't understand why we use `None` here instead of `False`. I use blame https://github.com/huggingface/transformers/blame/master/src/transformers/tokenization_bert.py#L610 and trace down to @n1t0 here. Could you explain? Thanks

Reply:
Anthony can chime in if I'm wrong, but I'm pretty sure it's related to [this](https://github.com/huggingface/tokenizers/blob/ee643812542b9a02c1bf03d84a499c16f71f49bc/bindings/python/py_src/tokenizers/normalizers/__init__.pyi#L34-L36).

It's set to `None` because if it was set to `True` or `False`, we wouldn't know if the user had explicitely specified a boolean value or if it was the default value. If it's the default value, the behavior should mimick that of the original BERT, which is to link `lowercase` and `strip_accents`.

Reply:
Ok I understand now. Thanks for your explanation.

Reply:
> Ok I understand now. Thanks for your explanation.

@JetRunner Is this PR ok with you?

Reply:
Sure! Let's merge it